src/main/java/org/eclipse/tractusx/managedidentitywallets/controller/DidDocumentController.java

```diff
@@ -109,6 +111,7 @@ public class DidDocumentController {
     @Operation(description = "Resolve the DID document for a given DID or BPN", summary = "Resolve DID Document")
     @GetMapping(path = RestURI.DID_DOCUMENTS, produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<DidDocument> getDidDocument(@Parameter(description = "Did or BPN",examples = {@ExampleObject(name = "bpn", value = "BPNL000000000000", description = "bpn"), @ExampleObject(description = "did", name = "did", value = "did:web:localhost:BPNL000000000000")}) @PathVariable(name = "identifier") String identifier) {
+        log.debug("Received Request to get DID document for identifier: {}", identifier);
```

Comment:
Let's keep it consistent
```suggestion
        log.debug("Received request to get DID document for identifier: {}", identifier);
```